Ports/poppler/package.sh

```diff
@@ -7,36 +7,41 @@ files=(
     "https://poppler.freedesktop.org/poppler-data-${version_data}.tar.gz#c835b640a40ce357e1b83666aabd95edffa24ddddd49b8daff63adb851cdab74"
 )
 depends=(
+    'boost'
     'curl'
     'fontconfig'
     'freetype'
+    'gpgme'
+    'lcms2'
     'libjpeg'
     'libpng'
     'libtiff'
+    'openjpeg'
 )
 useconfigure='true'
 configopts=(
     '-B build'
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
     '-DCMAKE_BUILD_TYPE=release'
-    '-DENABLE_BOOST=OFF'
     '-DENABLE_CPP=OFF'
     '-DBUILD_CPP_TESTS=OFF'
     '-DENABLE_GLIB=OFF'
     '-DENABLE_GOBJECT_INTROSPECTION=OFF'
-    '-DENABLE_GPGME=OFF'
-    '-DENABLE_GTK_DOC=OFF'
-    '-DENABLE_LCMS=OFF'
-    '-DENABLE_LIBOPENJPEG=unmaintained'
+    '-DENABLE_GPGME=OFF' # Enabling GPGME causes the program to crash (#23557)
     '-DENABLE_NSS3=OFF'
     '-DENABLE_QT5=OFF'
     '-DBUILD_QT5_TESTS=OFF'
     '-DENABLE_QT6=OFF'
     '-DBUILD_QT6_TESTS=OFF'
     '-DBUILD_GTK_TESTS=OFF'
-    '-DBUILD_MANUAL_TESTS=OFF'   
+    '-DBUILD_MANUAL_TESTS=OFF'
 )
 
+pre_configure() {
+    # Fix the GpgmeppConfig.cmake file to use the correct path for cross-compilation
```

Comment:
this looks like something we should fix in the install steps for the Gpgme port. Also since you've disabled GPEME in the CMakeLists, perhaps the preconfigure and post-install can be skipped entirely?

Reply:
How would you fix that in the gpgme install? The path in that file is absolutely correct for use on the target itself. It just does not work in our port system.

If we change that path in the gpgme install it will break on target usage.

What is true, that this is not needed at the moment because gpgme is disabled. I left it in for the moment we are able to enable gpgme again.

Reply:
wait, is the issue that the find_package() call in this application can't find gpgmepp? there should be automatic logic (in cmake itself?) to prefix paths with the CMAKE_SYSROOT from our CMakeToolchain.

If there's not, then patching the lib to prefix `${CMAKE_SYSROOT}` in the file should work just fine for both cross and native builds using that CMakeConfig file.

Reply:
This is the input:
```cmake
set_target_properties(Gpgmepp PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "@resolved_includedir@/gpgme++;@resolved_includedir@"
  INTERFACE_LINK_LIBRARIES "pthread;@resolved_libdir@/libgpgme@libsuffix@;@LIBASSUAN_LIBS@"
  IMPORTED_LOCATION "@resolved_libdir@/libgpgmepp@libsuffix@"
)
```
And this the currect output:
```cmake
set_target_properties(Gpgmepp PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "/usr/local/include/gpgme++;/usr/local/include"
  INTERFACE_LINK_LIBRARIES "pthread;/usr/local/lib/libgpgme.so;-lassuan -L/usr/local/lib -lgpg-error"
  IMPORTED_LOCATION "/usr/local/lib/libgpgmepp.so"
)
```

As this gets resolved at build time, I would need to change it after the output file is created.

Reply:
That works fine (and crashes poppler if we link against it)